cmd/admin-profile-stop.go

```diff
@@ -27,10 +27,10 @@ import (
 	"github.com/minio/mc/pkg/probe"
 )
 
-var adminProfilingStopCmd = cli.Command{
+var adminProfileStopCmd = cli.Command{
 	Name:            "stop",
-	Usage:           "Stop and download profiling data",
-	Action:          mainAdminProfilingStop,
+	Usage:           "Stop and download profile data",
```

Comment:
In previous `Usage:`s, everything was in lower case...
`Usage:  "Stop and download profiling data",` => `Usage:  "stop and download profiling data",`